src/types/chart/chart.ts

```diff
@@ -87,6 +87,7 @@ export interface TitleDesign {
   readonly italic?: boolean;
   readonly align?: Align;
   readonly color?: Color;
+  readonly fontSize?: number;
```

Comment:
Does it seems doable to make this property required ? ATM we need to do `??` condition every time we want to use the font size (a lot)

Reply:
Got it! I'll remove the optional mark from `TitleDesign`. You're right—using `??` every time we handle `fontSize` adds unnecessary overhead.